queries/ecma/highlights.scm

```diff
@@ -87,6 +90,9 @@
 (call_expression
   function: (member_expression
     property: (property_identifier) @method))
+(call_expression
+  function: (member_expression
+    property: (private_property_identifier) @method))
```

Comment:
I was wondering though. Is it possible to merge this into one? Like
```scheme
(call_expression
  function: (member_expression
    property: (property_identifier|private_property_identifier) @method))
```

Reply:
```
(call_expression
  function: (member_expression
    property: [(property_identifier) (private_property_identifier)] @method))
```

Reply:
Thanks @theHamsta. I'll update the PR shortly.

Reply:
We just added highlight tests to this repo. If you want to you can submit your example file with highlight assertions for the change (e.g. private fields that are now highlighted)

Format would be to add a caret below pointing the highlight that you changed
```ts
class {
  #priv_field = "World";
//   ^ property
}

```

This is not a must. I'm just  Your PR is awesome already!

Reply:
Sure, why not. I'll try to add test for this PR :)